lbrynet/daemon/Daemon.py

```diff
@@ -236,6 +239,8 @@ def _announce():
                 log.info("Started lbrynet-daemon")
                 log.info("%i blobs in manager", len(self.session.blob_manager.blobs))
 
+            if self.session.blob_manager is None:
+                return
```

Comment:
this will throw a deferred generator exception (returning from inlinecallbacks)

Reply:
also, why is this necessary?

Reply:
This change is part of this commit https://github.com/lbryio/lbry/pull/1007/commits/15fcd9a5f6df3aa1b534846c912003ccab2ff6f9
Basically I realized that when upnp fails and a shutdown is triggered in the middle of the startup, the blob_manager initialization is also stopped (as expected), but other routines still try to access it.
I added this check to prevent them from hitting the initialized object and explode.